src/cryptography/hazmat/primitives/twofactor/utils.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import unicode_literals
```

Comment:
For files we have a licensing preamble and a specific __future__ import we want: https://cryptography.io/en/latest/development/submitting-patches/#code